lib/plugins/aws/deploy/tests/deployFunctions.js

```diff
@@ -164,7 +164,7 @@ describe('deployFunctions', () => {
       serverless.config.servicePath = tmpDirPath;
 
       return awsDeploy.zipFunctions().then(() => {
-        expect(awsDeploy.deployedFunctions[0].zipFileData).to.be.not.empty;
+        expect(typeof awsDeploy.deployedFunctions[0].zipFileData).to.not.equal('undefined');
```

Comment:
`typeof` returns a string, so it's the same thing. This test makes sure that zipFileData is added ... it's exactly like the one u had before, but without the linting issue 😄